data/items/items.xml

```diff
@@ -2452,7 +2456,7 @@
 		<attribute key="slotType" value="ring" />
 		<attribute key="decayTo" value="0" />
 		<attribute key="transformDeEquipTo" value="2167" />
-		<attribute key="duration" value="600" />
+		<attribute key="duration" value="600000" />
```

Comment:
what is this? 😛

Reply:
a testing ring I forgot to remove. xD
Sorry >.<